searchlib/src/vespa/searchlib/diskindex/zcposocciterators.h

```diff
@@ -19,9 +19,10 @@ class ZcRareWordPosOccIterator : public ZcRareWordPostingIterator<bigEndian, dyn
     using DecodeContextReal = std::conditional_t<dynamic_k, bitcompression::EGPosOccDecodeContextCooked<bigEndian>, bitcompression::EG2PosOccDecodeContextCooked<bigEndian>>;
     DecodeContextReal _decodeContextReal;
 public:
-    ZcRareWordPosOccIterator(Position start, uint64_t bitLength, uint32_t docIdLimit, bool decode_cheap_features,
-                              const bitcompression::PosOccFieldsParams *fieldsParams,
-                              const fef::TermFieldMatchDataArray &matchData);
+    ZcRareWordPosOccIterator(Position start, uint64_t bitLength, uint32_t docIdLimit,
+                             bool decode_cheap_features, bool decode_normal_features,
```

Comment:
The order of the parameter list does not match the one in the cpp file. In the cpp file decode_normal_features comes before decode_cheap_features. We should have the same order here to avoid confusion.

Reply:
Fixed